01_score_even_vowels.py

```diff
@@ -0,0 +1,32 @@
+# get the score of each word using input from STDIN. Print the total score output to STDOUT
+# The score of a single word is 2 if the word contains an even number of vowels. Otherwise, the score of this word is 1 
+# vowels in the alphabet are a, e, i, o, u and y
+
+def score_words():
+    number_of_words = input("enter 2")
```

Comment:
Is this `number_of_words` doing anything in this function? Looks like this could be commented out.